manifests/cli_helper.pp

```diff
@@ -2,7 +2,16 @@
 #
 # A helper script for creating resources via the Jenkins cli
 #
-class jenkins::cli_helper {
+# Parameters:
+#
+# ssh_keyfile = undef
+#   Defaults to the value of $::jenkins::cli_ssh_keyfile. This parameter is
+#   deprecated, please set $::jenkins::cli_ssh_keyfile instead of setting this
+#   directly
+#
+class jenkins::cli_helper (
+  $ssh_keyfile = $::jenkins::cli_ssh_keyfile,
```

Comment:
I'm wondering if we need to parameterize this at all? Maybe `$::jenkins::cli_ssh_keyfile` should be the single user entry point and we can work towards making `jenkins::cli_helper` and internal only class?